app/build.gradle

```diff
@@ -121,6 +121,8 @@ dependencies {
     //Others
     compile 'org.jsoup:jsoup:1.7.3'
     compile 'joda-time:joda-time:2.3'
+    compile 'com.google.code.findbugs:jsr305:2.0.1'
```

Comment:
this is needed as the android gradle plugin uses a version for test and somehow it gets confused without this line.